src/Controller/ResultController.php

```diff
@@ -14,11 +15,21 @@
 class ResultController extends AbstractController
 {
     private $quizInstanceService;
-
-    public function __construct(RendererInterface $renderer, QuizInstanceService $quizInstanceService)
+    /**
+     * @var SessionInterface
+     */
+    private $session;
+    /**
+     * @var QuestionInstanceService
+     */
+    private $questionInstanceService;
+
```

Comment:
pls break parameters on multiple lines as such

`public function __construct(
        ScopeConfigInterface $scopeConfig,
        Session $checkoutSession,
        ItemConverter $converter
    ) {
        $this->checkoutSession = $checkoutSession;
        $this->converter = $converter;
        $this->scopeConfig = $scopeConfig;
    }`